ibis/backends/duckdb/__init__.py

```diff
@@ -987,7 +987,7 @@ def list_tables(
         return self._filter_with_like(out[col].to_pylist(), like)
 
     def read_postgres(
-        self, uri: str, table_name: str | None = None, schema: str = "public"
+        self, *, uri: str, table_name: str | None = None, schema: str = "public"
```

Comment:
Let's keep `uri` non-keyword.

```suggestion
        self, uri: str, *, table_name: str | None = None, schema: str = "public"
```

Reply:
Just to be sure, because in `read_mysql`, we made `uri` required, do we want to revert that? I think we should be consistent across the board.